shell/assets/translations/en-us.yaml

```diff
@@ -1744,6 +1744,7 @@ cluster:
     otccce: Open Telekom Cloud CCE
     oracle: Oracle
     oracleoke: Oracle OKE
+    ociocne: OCI OCNE
```

Comment:
Changes under shell need to be demarcated with the following marker comments:
// Added by Verrazzano Start
**** ... code changes ... ****
// Added by Verrazzano End